src/desktop/apps/home/client/index.coffee

```diff
@@ -27,13 +28,13 @@ module.exports.HomeView = class HomeView extends Backbone.View
     'blur #main-layout-search-bar-container': 'unhighlightSearch'
     'click #main-layout-search-bar-button': 'performSearch'
 
-  initialize: ->
+  initialize: ({ @testGroup }) ->
     # Set up a router for the /log_in /sign_up and /forgot routes
     new HomeAuthRouter
     Backbone.history.start pushState: true
 
     # Render Featured Sections
-    if sd.HIDE_HERO_UNITS then @setupSearchBar() else @setupHeroUnits()
+    if @testGroup == 'experiment' then @setupSearchBar() else @setupHeroUnits()
```

Comment:
#minor `if @testGroup is 'experiment'`

Reply:
Also, these are always accessible in `sd.HOME_PAGE_TEST == 'experiment'`, so you may not need any intermediate variable passing around.